PartitionedSolvers/src/smoothers.jl

```diff
@@ -65,166 +65,245 @@ function richardson(solver;iters,omega=1)
     linear_solver(;setup,update!,solve!,finalize!,step!,returns_history)
 end
 
-function jacobi(;kwargs...)
-    solver = jacobi_correction()
```

Comment:
Please use 4 spaces for indentation (do not use tabs!).

Follow the same indentation style as in the existing files.